1.7/administration/upgrading.md

```diff
@@ -9,6 +9,7 @@ This document provides instructions for upgrading a DC/OS cluster from version 1
 
 **Important:**
 
+- This document is currently in BETA status. Due to a known issue, this install method will cause all apps and tasks to restart within your cluster.
```

Comment:
I think this is an option as well. As I recall, @graham-murphy was interested in keeping it up, but I defer to him and Thomas to work this out.

I do concur that we will do better to never see this change published.